ansible/validation.yml

```diff
@@ -0,0 +1,99 @@
+---
+# A series of simple tests to validate the Slurm cluster deployment
+# These will eventually be split into different files to test the 
+# different stages of the deployment.
+# Login and compute groups of slurm cluster have been used as hosts for now.
+
+- hosts: openstack
+  roles:
+    - role: jasmin.cluster-infra
+      cluster_network: "caastest-U-internal"
+      cluster_deploy_user: 'centos'
+
+- hosts:
+  - login
+  - compute
+  tasks:
+    - name: get ids of all users
+      shell: id {{ item.username }}
+      with_items:
+        - "{{ jasmin_users }}" 
+      register: shell_result
+      run_once: true
+      ignore_errors: yes
+ 
+    - set_fact:
+        check_users: "{{ shell_result.results | selectattr('rc','equalto', 1) | map(attribute='stderr') | list}}"
```

Comment:
It would be better to catch all results where rc is not equal to zero, rather than select for equal to 1.